src/uct/api/uct.h

```diff
@@ -169,12 +172,14 @@ BEGIN_C_DECLS
  *      Open a connection manager.
  * @ref uct_ep_create
  *      Create an endpoint for establishing a connection to the server.
- * @ref uct_cm_ep_priv_data_pack_callback_t
- *      This callback is invoked by the UCT transport to fill the user's private data
- *      in the connection request to be sent to the server. This connection request
- *      should be created by the transport.
- *      Send the connection request to the server.
- *      Wait for an acknowledgment from the server, indicating that it is connected.
+ * @ref uct_cm_ep_resolve_callback_t
+ *      This callback is invoked by the UCT transport to provide addition
```

Comment:
This callback is invoked on the client side of the connection manager, after the remote server address was resolved to the local device to be used for connection establishment.